### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD015.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#21] [General]
# https://blog.3place.cn/3place/fsd/FSD015.html

https://blog.3place.cn/3place/fsd/FSD015.html

**Comment by picklecai** [comment]
您只要能想念我片刻工夫，
我将要一生一世将您怀念。

未经历世事的小姑娘，写得太真了。这个小姑娘，纯真，朴素，善良，容易信任别人，容易付出真心……每个姑娘，都有过这个阶段吧。只是其中有的人恰好16岁而已。

**Reply by picklecai** [reply]
这种状态，一旦过去了，是不可逆的。心理上“成年”后，再也养不出这样的状态来了。

**Comment by picklecai** [comment]
啊，我自己想，难道他看你的态
有点不大端庄，有点轻佻
他似乎立即引起这种想法
跟这个姑娘可以随便勾搭

每一头洁白的羔羊，都是先反思自己有无不妥。传统，就是这样教女儿的吧？
